moped-editor/src/queries/project.js

```diff
@@ -48,10 +48,12 @@ export const SUMMARY_QUERY = gql`
       moped_project {
         project_name
       }
-      moped_proj_components(where: { is_deleted: { _eq: false } }) {
-        moped_proj_features(where: { is_deleted: { _eq: false } }) {
-          feature_id
-          feature
+      moped_proj_components(
+        where: { is_deleted: { _eq: false }, feature_signals: {} }
+      ) {
+        feature_signals {
+          signal_id
+          knack_id
```

Comment:
oops—i need to touch this up with an `is_deleted` condition - in progress!

Reply:
✅  this now matches the same conditions we use in the [signals dashboard query](https://github.com/cityofaustin/atd-moped/blob/5eff270393e1e09553f7048bfa3ad92f09f8e3a5/moped-editor/src/queries/signals.js#L8)

the filter on `moped_proj_components` will only return project components that have at least one not-deleted signal. then another filter on `feature_signals` ensures that only not-deleted signals are returned.